selfdrive/ui/ui.hpp

```diff
@@ -112,12 +112,14 @@ typedef struct UIScene {
   // responsive layout
   Rect viz_rect;
 
-  std::string alert_text1;
-  std::string alert_text2;
-  std::string alert_type;
-  float alert_blinking_rate;
-  cereal::ControlsState::AlertSize alert_size;
-
+  struct Alert {
+    std::string type, text1, text2;
+    cereal::ControlsState::AlertSize size;
+    cereal::ControlsState::AlertStatus status;
+    AudibleAlert sound;
+    float blinking_rate = 0.;
+  } alert;
```

Comment:
why do we need this? we already save the controlsState packet

Reply:
It's is not read-only, UI will update this data structure when constrls&camera timeout, I think using controlstate can be more complicated than a struct